nengo_gui/tests/test_data_to_csv.py

```diff
@@ -0,0 +1,59 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver import ActionChains
+import time
+import pytest
+from nengo_gui import conftest
+from nengo_gui import testing_tools as tt
+import os
+
+def test_data_to_csv(driver):
+	tt.reset_page(driver)
+
+	tt.update_editor(driver,'''
+import nengo
```

Comment:
For multi-line strings, I would personally prefer if you did string concatenation like [Brent does here](https://github.com/nengo/nengo_gui/pull/663/files#diff-25dd944edac72587df925012e2e3d6f6R168). It sets off fewer "indentation alarms" when I'm reading code.